source/plugins/idea.rst

```diff
@@ -1,10 +1,36 @@
-=============
+=======
 IntelliJ IDEA
-=============
+=======
 
+.. tip::
+    IntelliJ is recomended to use by the Sponge team, as it is a very smart IDE for beginners and advanced users.
 
-You can download IntelliJ IDEA here: https://www.jetbrains.com/idea/download/
+Intellij for Java
+~~~~~~~~~~~~~~~~
+Intellij is a development environment that you can learn and code Java with. 
+*It is recommended that you have some experience with Java before you proceed to writing plugins in Sponge.*
 
+Downloads
+~~~~~~~~~
+**Download Intellij IDEA at:**
+https://www.jetbrains.com/idea/download/
 
-(Choose the "Community Version" unless you want to pay for the Ultimate Edition, or you can get it free as a student.
```

Comment:
Please do not remove this warning. Whilst I like and use IntelliJ it would be wrong not to tell beginners that it is commercial software, even if it largely doesn't matter for sponge development.

It could be worded better however.